controllers/jenkins/pipelinerun/pipelinerun_controller.go

```diff
@@ -309,6 +323,24 @@ func (r *Reconciler) makePipelineRunOrphan(ctx context.Context, pr *v1alpha3.Pip
 	return r.updateStatus(ctx, &pr.Status, client.ObjectKey{Namespace: pr.Namespace, Name: pr.Name})
 }
 
+func (r *Reconciler) getOrCreateJenkinsCoreIfHasCreator(annotations map[string]string) (*core.JenkinsCore, error) {
+	creator, ok := annotations[v1alpha3.PipelineRunCreatorAnnoKey]
+	if !ok || creator == "" {
+		return &r.JenkinsCore, nil
+	}
+	// create a new JenkinsCore for current creator
+	accessToken, err := r.TokenIssuer.IssueTo(&user.DefaultInfo{Name: creator}, token.AccessToken, tokenExpireIn)
+	if err != nil {
+		return nil, fmt.Errorf("failed to issue access token for creator %s, error = %v", creator, err)
```

Comment:
Please try to keep using the same style at least for your own code lines.

`error = %v`

[line 184](https://github.com/kubesphere/ks-devops/pull/384/files#diff-e19db2bfc951270073311530a9c79cb225bad3986a778c4decefd366e853880bR184) `error was %s`